modules/mono/glue/GodotSharp/GodotSharp/Core/Callable.generics.cs

```diff
@@ -0,0 +1,475 @@
+using System;
+using Godot.NativeInterop;
+
+namespace Godot;
+
+public readonly partial struct Callable
+{
+    /// <summary>
+    /// Constructs a new <see cref="Callable"/> for the given <paramref name="action"/>.
+    /// </summary>
+    /// <param name="action">Action method that will be called.</param>
+    public static unsafe Callable From(
+        Action action
+    )
+    {
+        static void Trampoline(object delegateObj, NativeVariantPtrArgs args, out godot_variant ret)
```

Comment:
Shouldn't we check the `args.Count` is 0 in this trampoline and the one for `Func<TResult>`?